__tests__/app.test.js

```diff
@@ -13,110 +13,135 @@ afterAll(() => {
             .then(({ body }) => {
-                expect(body.article).toStrictEqual(
-                    expect.objectContaining(
-                        {
-                            article_id: 3,
-                            title: "Eight pug gifs that remind me of mitch",
-                            topic: "mitch",
-                            author: "icellusedkars",
-                            body: "some gifs",
-                            created_at: "2020-11-03T09:12:00.000Z",
-                            votes: 0,
-                          }
-                    )
-                )
+                expect(body.comments).toStrictEqual([])
             })
+        });
     });
-});
-describe('GET /api/articles/:article_id/comments', () => {
-    it('should return a status 200 and an array of comment objects sorted by most recent comment first(descending order)', () => {
-        return request(app)
-            .get("/api/articles/9/comments")
-            .expect(200)
-            .then(({ body }) => {
-                expect(body.comments).toHaveLength(2)
-                expect(body.comments).toBeSortedBy("created_at", {
-                    descending: true
-                })
-                body.comments.forEach((comment) => {
-                    expect(comment).toStrictEqual(
+    describe('POST /api/articles/:article_id/comments', () => {
+        it('should return a status 201 and a comment object containing the new comment', () => {
+            const newComment = {
+                username: "rogersop",
+                body: "I am posting this comment to this api to see if it works."
+            }
+            return request(app)
+                .post("/api/articles/1/comments")
+                .send(newComment)
+                .expect(201)
+                .then(({ body }) => {
+                    expect(body.comment).toStrictEqual(
                         expect.objectContaining({
                             comment_id: expect.any(Number),
-                            votes: expect.any(Number),
-                            created_at: expect.any(String),
-                            author: expect.any(String),
-                            body: expect.any(String),
+                            body: "I am posting this comment to this api to see if it works.",
+                            votes: 0, 
+                            author: "rogersop",
+                            article_id: 1,
+                            created_at: expect.any(String)
                         })
                     )
                 })
-            })
-    });
-    it('should return a status 200 and an empty array if a valid, existent ID is provided but no comments are associated with that article', () => {
-    return request(app)
-        .get("/api/articles/2/comments")
-        .expect(200)
-        .then(({ body }) => {
-            expect(body.comments).toStrictEqual([])
-        })
+        });

```

Comment:
We can add an extra happy path for when there are extra properties in the object we are sending i.e. 
{ 
username: "rogersop",
body: "I am posting this comment to this api to see if it works."
banana: "fruit" 
}

This should ignore the extra key and just return a 201 🙂